junebug/api.py

```diff
@@ -77,14 +94,29 @@ def get_channel_list(self, request):
             },
             'required': ['type', 'config', 'mo_url'],
         }))
+    @inlineCallbacks
     def create_channel(self, request, body):
         '''Create a channel'''
-        raise NotImplementedError()
+        redis = yield self._get_redis_manager()
+        channel = Channel(
+            redis, self.amqp_config, body)
+        yield channel.save()
+        yield channel.start(self.service)
+        yield redis.close_manager()
```

Comment:
If we close the manager each request, what happens on the next request that needs the manager?